tools/content-generator/src/intTest/java/org/projectnessie/tools/contentgenerator/ITDeleteContent.java

```diff
@@ -43,7 +44,7 @@ void setup() throws NessieConflictException, NessieNotFoundException {
   void deleteContent() throws NessieNotFoundException {
 
     ProcessResult proc =
-        runGeneratorCmd(
+        RunContentGenerator.runGeneratorCmd(
```

Comment:
nit: Could we static-import stuff from `RunContentGenerator` and avoid these changes?